config/linux_gpu.cmake

```diff
@@ -33,6 +33,14 @@
 # Specify `cmake --build . --parallel N` to set the number of parallel compilation jobs.
 # Default is derived from CPUs available.
 #
+# By default, cmake will try and discover which GPU architecture to use by looking at
+# the available GPUs on the machine that you're building on. If you want to build for
+# a specific GPU architecture or are building on a machine without a GPU, then
+# specify the MXNET_CUDA_ARCH option like:
+#
+# $ cmake .. -DMXNET_CUDA_ARCH=7.0
+#
+# In the example above we're building for sm_70 which is the Volta architecture.
```

Comment:
@leezu @szha heres the doc changes we discussed offline for GPU builds specifying the architecture manually instead of auto discovering

Reply:
To avoid duplication, should we just reference the option and explanation 20 lines below.If you find the explanation below isn't clear, feel free to improve it. It may be better than documenting it twice. WDYT?

Reply:
lol, ya i totally didnt see that below. Plus I dont think I would have understood what it was saying. I needed a dumbed down version like "heres how to compile for a specific GPU...". 

Do we expect users will just build on the machine with the GPU that they intend to use? So the simple `cmake ..` is all they need? Who are these instructions for? If they were for me, they would say heres how to build for a specific GPU :D